### PR TITLE
[api] fix project kind removal

### PR DESCRIPTION
**Pull request by adrianschroeter**
...again....

### CODE REVIEW

File: src/api/app/models/project/update_from_xml_command.rb [hunk 1]
```diff
@@ -28,7 +28,8 @@ def run(xmlhash, force = nil)
       project.remoteurl = xmlhash.value('remoteurl')
       project.remoteproject = xmlhash.value('remoteproject')
       project.scmsync = xmlhash.value('scmsync')
-      project.kind = xmlhash.value('kind') if xmlhash.value('kind').present?
+      project.kind = xmlhash.value('kind').presence || 'standard'
```
Comment (eduardoj) [inline src/api/app/models/project/update_from_xml_command.rb:31]: I've searched backwards in the commit history about this change. The title of the pull request, the commit title and the description of the commit gave me a hint about fixing something, or reverting some change. I found these changes:

https://github.com/openSUSE/open-build-service/commit/c03f3e0ebbc2e97714ba684a71b8dd14a00e9578#diff-40f55459121e262fa08c43869c11eb281f30a3b5e0624fa8aee3f3e970e103f5L441-R451

Shouldn't we describe better why `'standard'` should be the default value for the `kind` of a project?

File: src/api/test/unit/project_test.rb [hunk 2]
```diff
@@ -372,6 +372,29 @@ def test_handle_project_links
...
+    xml = prj.to_axml
+    assert_xml_tag xml, tag: :project, attributes: { kind: 'maintenance_release' }
+
+    prj.update_from_xml!(Xmlhash.parse(
+                           "<project name='kind'>
+                             <title/>
+                             <description/>
+                           </project>"
+                         ))
+    assert_no_xml_tag xml, tag: :project, attributes: { kind: 'maintenance_release' }
```
Comment (eduardoj) [inline src/api/test/unit/project_test.rb:395]:
```suggestion
    prj.update_from_xml!(Xmlhash.parse(
                           "<project name='kind'>
                             <title/>
                             <description/>
                           </project>"
                         ))
    xml = prj.to_axml
    assert_no_xml_tag xml, tag: :project, attributes: { kind: 'maintenance_release' }
    assert_xml_tag xml, tag: :project, attributes: { kind: 'standard' }
```

If removing the kind of a project makes its new kind to be `'standard'`, shouldn't we check for standard too?

Comment (eduardoj) [reply src/api/test/unit/project_test.rb:395]: The minitests fail. I think there is an assignment missing in the test:

```
xml = prj.to_axml
```

**Comment by hennevogel** [comment]
`standard` is the default value in the database since 2015 

https://github.com/openSUSE/open-build-service/commit/c03f3e0ebbc2e97714ba684a71b8dd14a00e9578

**Comment by adrianschroeter** [comment]
and what has this to do with the issue?

this is about a different set kind and when the user removes the attribute from xml.

**Review by eduardoj — APPROVED** [review]

**Pull request merged**